projects/cashmere/src/lib/input/input.directive.ts

```diff
@@ -34,6 +34,9 @@ export class InputDirective extends HcFormControlComponent implements DoCheck {
 
     _componentId = this._uniqueInputId;
 
+
+    @Input() toggleVis: boolean;
```

Comment:
My preference would be to spell the variable name all the way out `toggleVisible`. Since it is an @Input we should add a comment to it so that the auto generated documentation will include it.

Reply:
It also looks like you need to run a `npm run lint` on the project and commit the changes.